README.md

```diff
@@ -18,6 +18,7 @@ A command line tool that helps in splitting webp animation into images
  - cmake
  - OpenCV 3
  - libwebp
+ - hdf5
```

Comment:
If you observe the Linux Github Runner, it successfully builds there without an explicit installation of hdf5.
I think it automatically gets installed with OpenCV.
Maybe hdf5 was a missing dependency particular to your system.

Reply:
You're right. Technically, hdf5 is required by openCV, but for my part it didn't get installed automatically. Maybe it should be added as a note?

Reply:
Yes. We can write it as a note.
Did you install opencv using pacman?

Reply:
I did install it with pacman. OpenCV installed libopencv_hdf.so, but that one got linker errrors without manually installing hdf5. Weird!